### PR TITLE
feat: Expose revision_history_limit attribute

### PR DESCRIPTION
**Pull request by therealrawal**
Closes https://github.com/lacework/terraform-kubernetes-agent/issues/32
JIRA https://lacework.atlassian.net/browse/ALLY-599

### CODE REVIEW

**Comment by afiune** [comment]
Make it so! Lets run our compatibility matrix 💯

tenor-137076186

File: main.tf [hunk 1]
```diff
@@ -29,8 +29,9 @@ resource "kubernetes_secret" "lacework_config" {
 
 resource "kubernetes_daemonset" "lacework_datacollector" {
   metadata {
-    name      = var.lacework_agent_name
-    namespace = var.namespace
+    name                   = var.lacework_agent_name
+    namespace              = var.namespace
+    revision_history_limit = var.revision_history_limit
```
Comment (afiune) [inline main.tf:34]: I think this attribute belongs to the `spec` block below, besides the `selector` block

Screen Shot 2021-08-09 at 8 25 37 AM

https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/daemonset#spec

**Review by afiune — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (therealrawal) [reply main.tf]: done, moved it under spec

**Comment by afiune** [comment]
MAKE_IT_SO 👍🏽

**Review by afiune — APPROVED** [review]
tenor-215483413

**Pull request merged**